lib/PrivateBin.php

```diff
@@ -373,6 +376,11 @@ private function _read($dataid)
                     unset($data->meta->salt);
                 }
                 $this->_data = json_encode($data);
+
```

Comment:
Please move this logic into the Paste object. The controller should not contain model logic.